src/greeter/UserModel.cpp

```diff
@@ -115,6 +123,17 @@ namespace SDDM {
 
         endpwent();
 
+        if (mainConfig.Users.ShowSavedUsers.get()) {
```

Comment:
I don't think it's worth doing this. Because the `lastUser` code above implies that the loading of the full model (`needAllUsers`) is disabled in the theme settings and we have exceeded the value of the avatar threshold.

But the new `ShowSavedUsers` option should be useful not only for this case, but also for the case when the loading of the entire model is enabled in the theme settings. For example, by default, the `SSSD` service disables enumeration (the `enumeration` option) of domain users in the general enumeration `getent passwd`, however, leaving the possibility to receive information about such an user through the `getent passwd username'.

So, with `needAllUsers` enabled and `ShowSavedUsers` enabled, we can display all local users and those domain users who logged in to this computer.